### ISSUE TITLE
Select parent only

### ISSUE THREAD

**Issue by mirhat** [alexandernst/angular-multi-select#110]
Is it possible to select parent only without propagating to children?

**Comment by alexandernst** [comment]
No, this is currently not possible. Nevertheless I might implement it. What is the use case for this?

**Comment by mirhat** [comment]
If You have some kind of hierarchy (like goverment) where parent is independent of children.

**Comment by alexandernst** [comment]
Let me go through some questions.

Let's say that you have a hierarchy such as:

    A (0) x
        a x
        b x
        c )0) x
            c1 x
            c2 x
        d x
    B x
    C x
    D (0) x
        a
        b

What would you expect to happen when you click/check `A`? Should it say `A (1) v`? Or maybe `A (0) v`?

**Comment by mirhat** [comment]
When you click on A it would turn to checked but a, b,c would all be unchecked.

**Comment by alexandernst** [comment]
Yes, that I understand, but what would the counter say? Would it say `1` or would it say `0`?

**Comment by mirhat** [comment]
In my case counter is not required, but in general I am not sure what it should be

**Comment by jonasmagas** [comment]
Solved?

**Comment by alexandernst** [comment]
not yet, I haven't had time to work on this.

**Comment by jonasmagas** [comment]
too bad! You did a good job so far
